modules/common/src/main/scala/surge/streams/EventSink.scala

```diff
@@ -20,11 +22,29 @@ trait EventHandler[Event] {
 }
 
 trait EventSink[Event] extends EventHandler[Event] {
+  val tracer: Tracer
```

Comment:
For consistency with what we expect the user to provide on the command engine side we should make this an instance of OpenTelemetry and create the tracer ourselves like we do there

Additionally I think it'd be good to default to no-op so users don't need to think about it out of the box if they're not using tracing

Reply:
You are correct